input/pagecontent/ig_licensing.md

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
@costateixeira can we remove this file?

Reply:
we can. I think this page was added to explain the different considerations for all IGs (not only WHO). we can leave it out for now?

Reply:
@ritikarawlani